sidebars.js

```diff
@@ -977,7 +977,10 @@ module.exports = {
       "Zeebe Gateway": ["self-managed/zeebe-gateway-deployment/zeebe-gateway"],
     },
     {
-      "Backup & Restore": ["self-managed/backup-restore/backup-and-restore"],
+      "Backup & Restore": [
+        "self-managed/backup-restore/backup-and-restore",
+        "self-managed/backup-restore/optimize-backup",
+      ],
```

Comment:
Due to the fact that we now have separate versions for Optimize than the rest of the docs, parallel sidebar changes will need to be made at https://github.com/camunda/camunda-platform-docs/blob/main/optimize_sidebars.js#L1540.